msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CreateBindingResourcePackageBase.cs

```diff
@@ -34,18 +33,63 @@ public override bool Execute ()
 				return false;
 			}
 
-			string bindingResourcePath = Path.Combine (ProjectDir, OutputPath, Path.ChangeExtension (Path.GetFileName (BindingAssembly), ".resources"));
-			Log.LogMessage (MSBStrings.M0121, bindingResourcePath);
+			var compress = false;
+			if (string.Equals (Compress, "true", StringComparison.OrdinalIgnoreCase)) {
```

Comment:
Maybe we should use https://github.com/xamarin/xamarin-macios/blob/main/msbuild/Xamarin.MacDev.Tasks.Core/Tasks/ParseBundlerArgumentsTaskBase.cs#L176 here since it supports a whole bunch of other strings beyond true/false?

If not, then this is fine (since we hard code settings in another file here)...

Reply:
MSBuild typically uses 'true' and 'false' (case insensitively), I just added 'auto' to the mix.